vhdbuilder/packer/install-dependencies.sh

```diff
@@ -561,10 +561,14 @@ fi
 # Please do not use the .1 suffix, because that's only for the base image patches
 # regular version >= v1.17.0 or hotfixes >= 20211009 has arm64 binaries.
 KUBE_BINARY_VERSIONS="$(jq -r .kubernetes.versions[] manifest.json)"
+KUBE_URL="$(jq -r .kubernetes.downloadURL manifest.json)"
```

Comment:
just food for thought, we should have some validation which makes sure the manifest file includes exactly what we expect, since if there isn't an entry for `.kubernetes.downloadURL`,  this would break.

we can probably bake that validation directly into the cue schema itself, if it's not already there

Reply:
Good point.  Will add that.

Reply:
I checked and the cue schema already requires it.